docs/node-operators/hot-cold-block-production.mdx

```diff
@@ -1,58 +1,80 @@
+### Configure your cold wallet
+
+Your cold wallet must be present in the consensus ledger before its stake can be counted when delegated to your hot wallet. 
+
+- If your cold wallet address is present in the genesis ledger, then no further action is needed. 
+- If not, you must acquire enough MINA tokens to meaningfully participate in consensus and send those tokens to this cold wallet address.
 
 ### Delegate your cold wallet to your hot wallet
 
-Next, ensure that you have delegated your cold account to your hot one. If you've set up your account to delegate to the correct spot in the genesis ledger, then no further action is needed. Otherwise you can delegate by using the `mina client delegate-stake` command if your key was generated on a computer. Otherwise, see the delegate instructions on the [Mina ledger README](https://github.com/jspada/ledger-app-mina/tree/v1.0.0-beta.2) to learn more.
+Next, ensure that you have delegated your cold wallet account to your hot wallet account. 
+
+- If you've set up your account to delegate to the correct spot in the genesis ledger, then no further action is needed. 
+- If not, you can delegate by using the `mina client delegate-stake` command if your key was generated on a computer. Otherwise, follow the delegate instructions on the [Mina ledger README](https://github.com/jspada/ledger-app-mina/tree/v1.0.0-beta.2).
 
 ## Block Production
 
-Follow the "Start up a node" instructions in the [Connecting to the Network](connecting-to-the-network) page using your hot wallet key.
+Follow the start up a node instructions in [Connecting to the Network](connecting-to-the-network) using your hot wallet key.
 
 ## Protocol Details
 
 You may be thinking: Why can't I create blocks directly from a secure enclave or other HSM device in Mina's protocol?
 
-Two components of block production require use of the private key in interesting ways: Winning a slot and creating the Blockchain SNARK. Creating the SNARK within the secure hardware especially is extremely difficult and likely impossible to do quickly enough for our protocol given today's technology.
+Two components of block production require using a private key in interesting ways: 
+
+- Winning a slot
+- Creating the blockchain SNARK
+
+Creating the SNARK within the secure hardware is extremely difficult and likely impossible to do quickly enough for the protocol with today's technology.
+
+### Winning a slot (out of the SNARK)
 
-### Winning a slot (out of the snark)
+A block producer can determine if they've won a slot by evaluating a VRF ( verifiable random function) using their private key. The VRF needs to be evaluated for your account and every account that delegates to you for all slots within an epoch. To our knowledge, no one has ever implemented a VRF on an HSM, but this is likely not too difficult.
```

Comment:
What is an HSM?

Reply:
Additionally, I don't understand this article's focus on HSMs, as it is irrelevant to our chain. It makes this come off as if we are saying "we are secure because there are not HSMs", but that is very misleading, because we would still be secure even if we there existed HSMs. I would recommend just taking out all the HSM details here, as it's not clear why it's in this section of the docs, and reads as if we are concerned about HSMs existing for some reason.

Reply:
I like your suggestion of getting a new PR for additional improvements, so I'll defer more updates from this convo

Reply:
I've removed HSM references, not sure where they originated